src/views/IndividualTeamView.vue

```diff
@@ -186,8 +181,40 @@ export default {
         // eslint-disable-next-line
         console.log(response);
         return getTeam(this.$route.params.id);
-      }).then((response2) => {
-        this.team = response2.data;
+      }).then((team) => {
+        this.team = team.data;
+        this.loaded = true;
+      }).catch((error) => {
+        // eslint-disable-next-line
+        console.log(error.message);
+      });
+    },
+    confirmTransfer(member) {
+      this.$bvModal.msgBoxConfirm(`Are you sure you would like to make ${member.username} team leader?`, {
+        size: 'sm',
+        okVariant: 'success',
+        cancelVariant: 'danger',
+        okTitle: 'Yes',
+        cancelTitle: 'No',
+        footerClass: 'p-2 border-top-0',
+        centered: true,
+      }).then((confirmed) => {
+        if (confirmed) {
+          return this.makeLeader({ newLeaderId: member.id });
+        }
+        return '';
+      }).catch((error) => {
+        // eslint-disable-next-line
+        console.log(error.message);
```

Comment:
At some point, we will have to replace these with errors the user sees, but it's probably best to do that all at once? 

cc @liammoyn

Reply:
I can make a separate ticket for someone to go through and display error messages from every back end call. It'll probably be a lot of copy paste anyway.

Reply:
Yep separate ticket is the way to go. Keep things consistent